### PR TITLE
Add base styling for inline and fenced code blocks

### PR DESCRIPTION
**Pull request by jimmynotjim**
The project doesn't currently style code blocks in any way. This adds sensible defaults that can easily be overridden through variables.

## Additions

- Added base styling for inline and fenced code blocks

## Testing

- This is difficult to test in any of our doc sites due to the existing code styling that lives in those. 

## Review

- @Scotchester 
- @cfarm 
- @anselmbradford 
- @sebworks 

## Screenshots

screen shot 2017-05-02 at 12 52 34 pm

screen shot 2017-05-02 at 12 58 07 pm

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by Scotchester** [comment]
That inline example looks a lot bigger than 13px. Is that actually how it renders out?

File: src/cf-core/src/cf-base.less [hunk 1]
```diff
@@ -539,5 +539,38 @@ figure {
...
+    background: @code-bg;
+    border-radius: 4px;
+    color: @code-text;
+    font-family: 'Input Mono', Consolas, Monaco, 'Courier New', monospace;
+}
+
+code {
+    display: inline-block;
+    padding: unit( 3px / @size-code, em )
+             unit( 6px / @size-code, em )
```
Comment (Scotchester) [inline src/cf-core/src/cf-base.less]: This side padding looks a bit too generous, to my eyes, but maybe it's just me.

File: src/cf-core/src/cf-base.less [hunk 2]
```diff
@@ -539,5 +539,38 @@ figure {
...
+             0;
```
Comment (Scotchester) [inline src/cf-core/src/cf-base.less]: Having 0 bottom padding doesn't feel right. What happens if one of the letters has a descender? Does it get uncomfortably close to the bottom edge?

**Comment by jimmynotjim** [comment]
screen shot 2017-05-16 at 10 40 27 am

@Scotchester I've removed all the custom code styles on the GH pages branch to show what these defaults look like, I think a few were still getting through in the above screenshots.

- The text size looks correct, I'm assuming the previous screenshots were affected by the GH pages branch styles.
- Agreed on the side padding, I've reduced it to 3px to match the top.
- I think the bottom space is ok, the problem with adding bottom padding is it'll mess up the line-height if the inline code has a line of text below it, like in a paragraph.

**Comment by Scotchester** [comment]
:+1: :shipit:

**Pull request merged**